docs/README.rst

```diff
@@ -4,7 +4,7 @@ Documentation for boto3 can be found `here <https://boto3.amazonaws.com/v1/docum
 
 Generating Documentation
 ~~~~~~~~~~~~~~~~~~~~~~~~
-Note: Botocore's `requirement-docs.txt <https://github.com/boto/botocore/blob/develop/requirements-docs.txt>`_ must be installed prior to attempting the following steps.
+Note: Boto3's `requirements-docs.txt <https://github.com/boto/boto3/blob/develop/requirements-docs.txt>`_ must be installed prior to attempting the following steps.
```

Comment:
I'm not sure this is the right change. We're telling users they need to install the Botocore doc requirements before the boto3 doc requirements because they are both needed.

Reply:
OK will remove for the time being until I can confirm the right install directions.